src/physfs.cpp

```diff
@@ -103,289 +105,304 @@ base_fstream::base_fstream(PHYSFS_File* file) : file(file) {
 
 sint64 Util::swapSBE64(sint64 value) {
-	return PHYSFS_swapSBE64(value);
+    return PHYSFS_swapSBE64(value);
 }
 
 uint64 Util::swapUBE64(uint64 value) {
-	return PHYSFS_swapUBE64(value);
-}
-
-string Util::utf8FromUcs4(const uint32* src) {
-	string value;
-	std::size_t length = strlen((char*) src);
-	char * buffer = new char[length]; // will be smaller than len
-	PHYSFS_utf8FromUcs4(src, buffer, length);
-	value.append(buffer);
-	return value;
-}
-
-string Util::utf8ToUcs4(const char* src) {
-	string value;
-	std::size_t length = strlen(src) * 4;
-	char * buffer = new char[length]; // will be smaller than len
-	PHYSFS_utf8ToUcs4(src, (uint32*) buffer, length);
-	value.append(buffer);
-	return value;
-}
-
-string Util::utf8FromUcs2(const uint16* src) {
-	string value;
-	std::size_t length = strlen((char*) src);
-	char * buffer = new char[length]; // will be smaller than len
-	PHYSFS_utf8FromUcs2(src, buffer, length);
-	value.append(buffer);
-	return value;
-}
-
-string Util::utf8ToUcs2(const char* src) {
-	string value;
-	std::size_t length = strlen(src) * 2;
-	char * buffer = new char[length]; // will be smaller than len
-	PHYSFS_utf8ToUcs2(src, (uint16*) buffer, length);
-	value.append(buffer);
-	return value;
-}
-
-string Util::utf8FromLatin1(const char* src) {
-	string value;
-	std::size_t length = strlen((char*) src) * 2;
-	char * buffer = new char[length]; // will be smaller than len
-	PHYSFS_utf8FromLatin1(src, buffer, length);
-	value.append(buffer);
-	return value;
+    return PHYSFS_swapUBE64(value);
+}
+
+std::string Util::utf8FromUcs4(const uint32* src) {
+    std::string value;
+    std::size_t length = strlen((char*) src);
+    char * buffer = new char[length]; // will be smaller than len
+    PHYSFS_utf8FromUcs4(src, buffer, length);
+    value.append(buffer);
+    return value;
+}
+
+std::string Util::utf8ToUcs4(const char* src) {
+    std::string value;
+    std::size_t length = strlen(src) * 4;
+    char * buffer = new char[length]; // will be smaller than len
+    PHYSFS_utf8ToUcs4(src, (uint32*) buffer, length);
+    value.append(buffer);
+    return value;
+}
+
+std::string Util::utf8FromUcs2(const uint16* src) {
+    std::string value;
+    std::size_t length = strlen((char*) src);
+    char * buffer = new char[length]; // will be smaller than len
+    PHYSFS_utf8FromUcs2(src, buffer, length);
+    value.append(buffer);
+    return value;
+}
+
+std::string Util::utf8ToUcs2(const char* src) {
+    std::string value;
+    std::size_t length = strlen(src) * 2;
+    char * buffer = new char[length]; // will be smaller than len
+    PHYSFS_utf8ToUcs2(src, (uint16*) buffer, length);
+    value.append(buffer);
+    return value;
+}
+
+std::string Util::utf8FromLatin1(const char* src) {
+    std::string value;
+    std::size_t length = strlen((char*) src) * 2;
+    char * buffer = new char[length]; // will be smaller than len
+    PHYSFS_utf8FromLatin1(src, buffer, length);
+    value.append(buffer);
+    return value;
 }
 
 }
```

Comment:
Similar to my comment about the header file, I like these changes, and will include them verbatim, if you address other issues with this pull request or create a separate pull request.